Source/Charts/Utils/Platform+Touch Handling.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Platform+Touch Handling.swift
```

Comment:
I was thinking if we should have space in file name

Reply:
No reason not to